components/TARGET_PSA/TESTS/compliance_attestation/test_a001/main.c

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright (c) 2020, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Thanks, will create an issue today with the scan findings there

Reply:
I checked psa-arch-tests with scancode, no offenders found - all good there. These tests are only here?

Reply:
Looks like they've already added SPDX since our last import. There will be no issue with losing the identifiers. https://github.com/ARM-software/psa-arch-tests/blob/master/api-tests/dev_apis/initial_attestation/test_a001/test_a001.c#L3